src/discv5.rs

```diff
@@ -43,6 +46,40 @@ lazy_static! {
         RwLock::new(crate::PermitBanList::default());
 }
 
+/// Helper function that returns the hash of the given topic string according to the
+/// implemented hashing algorithm.
+pub static HASH: for<'a> fn(topic: &'a str) -> TopicHash = |topic| {
+    let sha256_topic = Topic::new(topic);
+    sha256_topic.hash()
+};
+
+// Discv5 versions.
+iota! {
+    pub const NAT: u8 = 1 << iota;
+        , TOPICS
+}
```

Comment:
I want to use bits so that it is easy to check if both versions are supported without making a third enum variant "NatAndTopics", how do you suggest I do this with an enum?

Reply:
I figured it out, thanks for pointing it out